core/lib/types/src/fee.rs

```diff
@@ -22,6 +22,9 @@ pub struct TransactionExecutionMetrics {
     pub total_log_queries: usize,
     pub cycles_used: u32,
     pub computational_gas_used: u32,
+    // Field holding the total number of bytes used in storage writes. Unused for now
+    // but required for new pubdata compression
+    pub total_updated_values_size: usize,
```

Comment:
nit: having a comment is sufficient but in theory we could name it  `updated_values_bytes_size`. Feel free to discard.

Reply:
okay, i think thats a fair point. if another update is needed before merging in then i will include this as well